stdlib/3.4/enum.pyi

```diff
@@ -20,8 +21,6 @@ class Enum:
 class IntEnum(int, Enum):
     value = ...  # type: int
 
-_T = TypeVar('_T')
-
 def unique(enumeration: _T) -> _T: ...
```

Comment:
Oh, hm, that might be by accident, or because something defaults to Any when mypy doesn't understand it. Would be nice to try this and see what the reveal_type(unique(Color)) actually says.